packages/react-components/src/components/EmptyState/EmptyState.tsx

```
@@ -0,0 +1,73 @@
+          )}
+          <Heading className={styles[`${baseClass}__title`]}>{title}</Heading>
+        </>
+      )}
+
+      <>
+        {type === 'full' && description && (
+          <Text noMargin className={styles[`${baseClass}__description`]}>
+            {description}
+          </Text>
+        )}
+      </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)